avocado/utils/multipath.py

```diff
@@ -215,9 +217,10 @@ def get_path_status(disk_path):
 
 def fail_path(path):
     """
-    failing the individual paths
-    :param disk_path: disk path. Example: sda, sdb.
-    :return: True or False
+    Fail the individual paths.
+
+    :param path: disk path. Example: sda, sdb.
+    :return: True if succeeded, False otherwise
```

Comment:
For instance something like this I would write like this (to minimize redundancy):
```
:returns: whether ... succeeded
:rtype: bool
```
I don't really understand what it means for a path to succeed or not but just visualizing here.

Reply:
I agree, but as I said, I will leave those improvements for future PRs.